hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptorRetry.java

```diff
@@ -150,8 +150,8 @@ protected YarnConfiguration createConfiguration() {
         mockPassThroughInterceptorClass + "," + mockPassThroughInterceptorClass
             + "," + TestableFederationClientInterceptor.class.getName());
 
-    conf.set(YarnConfiguration.FEDERATION_POLICY_MANAGER,
-        UniformBroadcastPolicyManager.class.getName());
```

Comment:
Do we have any tests now with UniformBroadcastPolicyManager?

Reply:
Thank you very much for helping to review the code!

`UniformBroadcastPolicyManager ` is the default policy, 
 if we do not set yarn.federation.policy-manager, We will use `UniformBroadcastPolicyManager` by default. 

`TestFederationClientInterceptor` will use `UniformBroadcastPolicyManager`.

I will modify the unit test code so that TestFederationClientInterceptorRetry overrides both `TestSequentialBroadcastPolicyManager`, `UniformBroadcastPolicyManager`.